identity-core/src/utils/ed25519.rs

```diff
@@ -11,6 +11,7 @@ use crate::error::Result;
 ///
 /// Note that the private key is a 32-byte seed in compliance with [RFC 8032](https://datatracker.ietf.org/doc/html/rfc8032#section-3.2).
 /// Other implementations often use another format. See [this blog post](https://blog.mozilla.org/warner/2011/11/29/ed25519-keys/) for further explanation.
+// TODO: move or remove
```

Comment:
Can we inline this into `KeyPair::new`, like how it is with `X25519` now?

Reply:
It's used for `KeyCollection` I think, so it will be removed with the `MerkleKeyCollection` removal.